### PR TITLE
Dropdown menu

### PR DESCRIPTION
**Pull request by rthurman56**
# PR Checklist
- [ ] Linting, tests, and coverage is passing when running `yarn test`

### CODE REVIEW

File: client/src/screens/Analytics.js [hunk 1]
```diff
@@ -14,20 +18,160 @@ export default class Analytics extends Component {
...
-                            <TwitterFollowerRatio {...this.props} />
-                        </Col>
-                    </Row>
-                </Container>
-            </div>
-        );
+
+        const { value } = this.state;
+
+        if(value == "Instagram"){
```
Comment (DominicSherman) [inline client/src/screens/Analytics.js]: Can you make `Instagram`, `Facebook`, and `Twitter` into variables that are just strings? Like we have our `actions`

File: client/src/screens/Analytics.js [hunk 2]
```diff
@@ -15,23 +18,160 @@ export default class Analytics extends Component {
...
     }
 
+    constructor(props) {
+        super(props);
+        this.toggle = this.toggle.bind(this);
+        this.select = this.select.bind(this);
+        this.state = {
+            dropdownOpen: false,
+            value : "Analytics View",
+            posts : [
```
Comment (DominicSherman) [inline client/src/screens/Analytics.js]: Do you use this `this.state.posts` anywhere?

File: client/src/screens/Analytics.js (hunk 1 above)
Comment (rthurman56) [reply client/src/screens/Analytics.js]: taken care of 👍

**Review by DominicSherman — APPROVED** [review]
Looks good! :shipit:

**Pull request merged**